tests/installation/partitioning_raid.pm

```diff
@@ -448,6 +439,11 @@ sub run {
     if (is_boot_raid_partition_required) {
         add_raid_boot;
     }
+}
+
+sub add_raid_swap {
+    # for newer storage-ng toolbar has changed
+    $cmd{addraid} = 'alt-d' unless $older_product;
```

Comment:
It's the same line in multiple places. Why not to move it at least to `run` subroutine, or even better to `tests/installation/partitioning.pm` where we redefine shortcuts already to have it in single place.

Reply:
I tried on top of the file, but it didn't work, Perl evaluate it before `$older_product`. I don't have scope of the code in `partitioning.pm` yet, so let's put it in `run` until the refactor in [poo#42401](https://progress.opensuse.org/issues/42401).